libs/features/mas/docs/benchmarks.html

```diff
@@ -0,0 +1,130 @@
+
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <title>M@S Benchmarks</title>
+  <script>
+    const start = performance.now();
+    window.commercePromise = new Promise((resolve) => {
+      document.addEventListener('wcms:commerce:ready', () => {
+        //we measure the time it takes to load the commerce bundle and store it in a global variable
+        const end = performance.now();
+        window.initTime = end - start;
+        console.log(`Commerce loaded in ${window.initTime}ms`);
+        resolve();
+      });
+    });
+  </script>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!-- Fonts -->
+    <link rel="stylesheet" href="https://use.typekit.net/hah7vzn.css">
+    <!-- Include any additional stylesheets -->
+    <link rel="stylesheet" href="styles.css">
+    <script type="module">
+      import { init } from './common.js';
+      init();
+    </script>
+</head>
+
+<script>
+  const measureCardPerformances = async (container, fragmentId) => {
```

Comment:
we should probably incorporate performance api to `merch-card` and `aem-fragment` custom elements directly.

and use performance api entries while reporting the measurements.